Makefile

```
@@ -29,9 +29,10 @@ tidy:
 
 # Install the tools for local development.
 install-tools:
-	go install github.com/deepmap/oapi-codegen/v2/cmd/oapi-codegen@latest
+	go install github.com/oapi-codegen/oapi-codegen/v2/cmd/oapi-codegen@latest
 	go install github.com/sqlc-dev/sqlc/cmd/sqlc@latest
 	go install ariga.io/atlas/cmd/atlas@latest
+	go install github.com/cosmtrek/air@v1.40.4
```

```suggestion
	go install github.com/air-verse/air@v1.40.4
```
Looks like the repo has moved to its own organization
Might make sense to reference its new name here instead

---

Committed the above :)